File: src/com/castlefrog/agl/domains/biniax/BiniaxAction.java
package com.castlefrog.agl.domains.biniax;

import com.castlefrog.agl.Action;

public enum BiniaxAction implements Action {
    NORTH,
    EAST,
    SOUTH,
    WEST;

    public BiniaxAction copy() {
        return this;
    }
}

File: src/main/kotlin/com/castlefrog/agl/domains/biniax/BiniaxAction.kt
package com.castlefrog.agl.domains.biniax

import com.castlefrog.agl.Action

enum class BiniaxAction : Action<BiniaxAction> {
    NORTH,
    EAST,
    SOUTH,
    WEST;

    override fun copy(): BiniaxAction = this
}
